test/test_helper.js

```diff
@@ -106,20 +106,39 @@ const startServer = (configFile = '../server/config.test.json') =>
     });
   });
 
-const createToken = (user) => {
+const createAuth0Token = (user) => {
   const options = {
     expiresIn: '5m',
     audience: config('AUTH0_CLIENT_ID'),
-    issuer: 'https://auth0.example.com'
+    issuer: `https://${config('AUTH0_DOMAIN')}/`
   };
 
   const userSub = {
     sub: user.user_id,
     email: user.email,
-    base: 'auth0.example.com/api/v2'
+    base: 'auth0.example.com/api/v2',
+    scope: []
   };
 
   return sign(userSub, config('AUTH0_CLIENT_SECRET'), options);
 };
 
-module.exports = { startServer, request: createRequest, createServer, createToken };
+const createWebtaskToken = (user) => {
```

Comment:
What to call this `createInvalidToken` instead? And just us respective constants / random values in the options below?

Reply:
That webtask token technically isn't an invalid token. It's used for dashboard admins

Reply:
To access the `/meta` endpoint, we need that webtask token

Reply:
Here is a test for that token: 
https://github.com/auth0-extensions/auth0-account-link-extension/blob/641bdf84b1da532a62b37c30a2551dba28b2f103/test/acceptance/endpoints_test.js#L30

Reply:
There are more tests on my node 22 upgrade PR that test each endpoint using a webtask token